### PR TITLE
Personnel overview

### PR DESCRIPTION
**Pull request by SvanSomeren**

### CODE REVIEW

File: src/pages/personnel/duck/Reducers.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+const defaultState = {
+    personnel: [],
+    clients: [],
+    errors: {},
+};
+
+const personnel = ( state = defaultState.personnel, { type, payload } ) => {
+    if ( type === types.LOAD_SUCCESS ) {
+        // call to get roles
```
Comment (tomdewildt) [inline src/pages/personnel/duck/Reducers.js]: Is this still relevant?

File: src/pages/personnel/duck/Sagas.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    }
+}
+
+function* create( { payload } ) {
+    try {
+        const { data } = yield call( axios.post, "/personnel/", payload );
+        yield put( { type: types.CREATE_SUCCESS, payload: data } );
+    } catch ( error ) {
+        yield put( {
+            type: errorTypes.APP_ERROR,
```
Comment (tomdewildt) [inline src/pages/personnel/duck/Sagas.js]: Use the error selector here (to display validation errors)

File: src/pages/personnel/person/Person.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        Load( match.params.id );
+    }
+
+    render() {
+        const {
+            person,
+            errors,
+            Edit,
+        } = this.props;
+        console.log( person );
```
Comment (tomdewildt) [inline src/pages/personnel/person/Person.js]: Remove this log

File: src/pages/personnel/person/duck/Reducers.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { combineReducers } from "redux";
+
+import types from "./Types";
+
+const defaultState = {
+    person: {},
```
Comment (tomdewildt) [inline src/pages/personnel/person/duck/Reducers.js]: Use null for default state and use the loading component in the page

File: src/pages/personnel/duck/Reducers.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import { combineReducers } from "redux";
+
+import types from "./Types";
+
+const defaultState = {
+    personnel: [],
+    clients: [],
```
Comment (tomdewildt) [inline src/pages/personnel/duck/Reducers.js]: Use null for personnel and clients default state and display loading page in page

File: src/pages/personnel/person/duck/Sagas.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+function* edit( { payload } ) {
+    const id = yield select( selectors.employeeId );
+    try {
+        const { data } = yield call( axios.patch, `/personnel/${ id }`, payload );
+        console.log( data );
+        yield put( { type: types.EDIT_SUCCESS, payload: data } );
+    } catch ( error ) {
+        yield put( {
+            type: errorTypes.APP_ERROR,
```
Comment (tomdewildt) [inline src/pages/personnel/person/duck/Sagas.js]: Use form error here with the error selector

File: src/pages/personnel/duck/Reducers.js (hunk 1 above)
Comment (SvanSomeren) [reply src/pages/personnel/duck/Reducers.js]: Dacht dat die al gone was. My bad

File: src/pages/personnel/duck/Sagas.js (hunk 2 above)
Comment (SvanSomeren) [reply src/pages/personnel/duck/Sagas.js]: Bij create of?? Want die erboven gebruikt em gwn. Zit op mobiel dus zie niet alles

**Review by tomdewildt — APPROVED** [review]

File: src/pages/personnel/person/Person.js (hunk 3 above)
Comment (liweihu1) [reply src/pages/personnel/person/Person.js]: console.log was removed

**Pull request merged**